pyfar/plot/line.py

```diff
@@ -16,7 +16,8 @@ def time(signal, dB=False, log_prefix=20, log_reference=1, unit=None, ax=None,
     Parameters
     ----------
     signal : Signal, TimeData
-        The input data to be plotted.
+        The input data to be plotted. Multidimensional data are flattened for
+        plotting.
```

Comment:
I think the statement `Multidimensional data are flattened for plotting.` is a little confusing, as it only means that all channels are shown. When cycling through the channels, the dimensions are somehow kept in the sense of multidimensional indexes. What's meant with this statement is that `Multidimensional data are flattened for plotting, so all channels are shown.`, isn't it? Maybe it can be formulated a little more precisely.
This also holds for the other changed docstrings.

Reply:
Good point. How about not mentioning this at all? One could argue that plotting all channels is the expected behavior and the detail that we internally flatten the signal to achieve this is not relevant for the user.

Reply:
I'd describe in which order the channels are plotted (and cycled). The used operation - which in this case is flattening - isn't necessarily meaningful to the users.

Reply:
Good point - I've left the mention of flattening and added an example of how multi-dimensional data would be plotted.